tools/parse-pkgs.sh

```diff
@@ -17,21 +17,36 @@ get_git_tag() {
 
 linuxkit_tag() {
     local pkg="$1"
-    _linuxkit_tag 0 "${pkg}"
+    _linuxkit_tag 0 0 "${pkg}"
 }
 
 linuxkit_dev_tag() {
     local pkg="$1"
-    _linuxkit_tag 1 "${pkg}"
+    _linuxkit_tag 0 1 "${pkg}"
+}
+
+linuxkit_kubevirt_tag() {
+    local pkg="$1"
+    _linuxkit_tag 1 0 "${pkg}"
+}
+
+linuxkit_kubevirt_dev_tag() {
+    local pkg="$1"
+    _linuxkit_tag 1 1 "${pkg}"
 }
 
 _linuxkit_tag() {
```

Comment:
Would it make sense to add a comment to document the three arguments and their types?